scripts/launcher/lib/log_handler.py

```diff
@@ -32,10 +32,12 @@ class VirtualLogRequestHandler(LogRequestHandler):
     simple_tests = LogRequestHandler.simple_tests + [
         "Payload setup error:",
         "Out of memory:",
-        "The following group or module is missing:",
+        "Would you like to ignore this and continue with installation?",
+        "Some packages, groups or modules are broken, the installation will be aborted.",
         "Stream was not specified for a module",
         "The following problem occurred on line",  # kickstart parsing error
         "storage configuration failed:",
+        "Disk Requirements:",  # payload does not fit into available storage
```

Comment:
This is really generic message. Couldn't this make trouble to us in the future?

Reply:
IIRC there was no better single line message to match on unfortunately. 
So I suggest to merge this now and then go change the message in Anaconda to something more distinct (like "Can't Satisfy Disk Requirements:"). Then once it bubbles it's way through on Fedora & RHEL we can update it here as well. :)

Reply:
Actually it looks like the message comes from DNF:
https://github.com/rpm-software-management/dnf/blob/e9f362facbaf999348e56b5f6a7bb4c245a1080a/dnf/base.py#L880

But this seems to be prefixed with "Error Summary", which could be a better match.